test/test_stoch_noise_sim.cpp

```diff
@@ -153,7 +153,7 @@ TEST(StochNoiseSimTest, ApproximateByFidelity) {
     auto quantumComputation = std::make_unique<qc::QuantumComputation>(3);
     quantumComputation->h(0);
     quantumComputation->h(1);
-    quantumComputation->emplace_back<qc::StandardOperation>(3, dd::Controls{dd::Control{0}, dd::Control{1}}, 2, qc::X);
+    quantumComputation->emplace_back<qc::StandardOperation>(3, qc::Controls{qc::Control{0}, qc::Control{1}}, 2, qc::X);

```

Comment:
actively use the literals and convenience functions to make this a little less ugly 😅